canvas/templates/canvas/course_registration.html

```diff
@@ -1,70 +1,54 @@
 {% extends 'base.html' %}
 
 {% block header %}
-    Courses Registration
+    Course Registration
 {% endblock %}
 
 {% block content %}
-    {% if blocked %}
```

Comment:
When I toggled the "is blocked" in CourseRegistrationObject, the page successfully showed only the alert message, but I definitely see how the current setup doesn't guarantee that. I have a commit to wrap the content with {% if not blocked %}, since the alert is taken care of by the messaging system